simplified-viewer-audiobook/src/main/java/org/nypl/simplified/viewer/audiobook/AudioBookPlayerActivity.kt

```diff
@@ -272,55 +272,6 @@ class AudioBookPlayerActivity :
     return entry.authors.first()
   }
 
-  override fun onDestroy() {
```

Comment:
Why is this being removed?

Reply:
Sorry, I should've included that on the description. It's being removed because thanks to the latest changes on the player events, it creates a remote bookmark when it's paused. So, if the _onDestroy_ was implemented it would create it twice because the player is paused before being destroyed.

Reply:
You're right, my mistake! I only meant to delete that part but somehow deleted everything. Reverting it now